samples/Samples.SqlServer/Properties/launchSettings.json

```diff
@@ -12,7 +12,9 @@
         "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
 
         "DD_DOTNET_TRACER_HOME": "$(ProjectDir)$(OutputPath)profiler-lib",
-        "DD_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json"
+        "DD_INTEGRATIONS": "$(ProjectDir)$(OutputPath)profiler-lib\\integrations.json",
+        "DD_AZURE_APP_SERVICES": "1",
+        "APP_POOL_ID": "~1WhyUNoWork"
```

Comment:
Could you put this in another profile? Also, can you put a comment here to remind us what the APP_POOL_ID setting is testing?